### PR TITLE
VACMS-3931: improve system menu ux

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #3931 

## Testing done

* spot-checked random menu items

## Screenshots

Screen Shot 2021-02-01 at 10 54 42 AM

## QA steps

1. Review updates and check for mistakes on VISN/paths

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by ndouglas — APPROVED** [review]
Spot-checked replacements and ran script locally, changes matched up 👍

**Comment by ndouglas** [comment]
Nice catch by either you or the Spec Tool 😁

**Review by ndouglas — APPROVED** [review]

File: config/sync/system.menu.va-southeast-louisiana-health-ca.yml [hunk 1]
```diff
@@ -10,5 +10,5 @@ third_party_settings:
       231: '231'
 id: va-southeast-louisiana-health-ca
 label: 'VA Southeast Louisiana health care'
-description: 'VISN 15 | va.gov/st-louis-health-care'
+description: 'VISN 16 | va.gov/va-southeast-louisiana-health-care'
```
Comment (ndouglas) [inline config/sync/system.menu.va-southeast-louisiana-health-ca.yml:13]: This shouldn't be `va.gov/southeast-louisiana-health-care`?

Comment (acrollet) [reply config/sync/system.menu.va-southeast-louisiana-health-ca.yml:13]: hmm, good question - a redirect exists for `southeast-louisiana-health-care` but the alias is set as `va-southeast-louisiana-health-care`: https://staging.cms.va.gov/node/7221/edit

**Comment by acrollet** [comment]
> Nice catch by either you or the Spec Tool 😁

the spec tool made me catch it 🙄

**Review by ndouglas — APPROVED** [review]

**Comment by ndouglas** [comment]
you set 'em up, and I'll knock 'em back 

tumblr_m20l06Zjku1r0sxi0o1_500

**Comment by acrollet** [comment]
> you set 'em up, and I'll knock 'em back
> 
> tumblr_m20l06Zjku1r0sxi0o1_500

**Pull request merged**